package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "broccoli-merge-trees": "^1.1.1",
-    "broccoli-persistent-filter": "1.2.0",
+    "broccoli-persistent-filter": "^1.0.0",
```

Comment:
I don't think it matters. I don't see how someone could install something less than 1.2.0.